### ISSUE TITLE
Releases page on WordPress.org references 6.0 branch being actively maintained

### ISSUE THREAD

**Issue by rkaiser0324** [WordPress/Documentation-Issue-Tracker#632]
## Issue Description
On the Releases page it says 
`None of these are safe to use, except the latest in the 6.0 series, which is actively maintained.`
I don't think that statement is completely accurate.

## URL of the Page with the Issue
https://wordpress.org/download/releases/ 

## Section of Page with the issue
n/a

## Why is this a problem?
It's worded awkwardly (and possibly inaccurate) and should be more precise and understandable. 

## Suggested Fix
Reword it to be something like:

`The only releases safe to use are the latest releases in the 6.X branches, which are actively maintained.`

I'm not sure what the actual policy is so maybe that's not accurate either - should be confirmed.

**Comment by zzap** [comment]
Heads up @WordPress/docs-issues-coordinators, we have a new issue open. Time to use 'em labels.

**Comment by estelaris** [comment]
the statement on the download page is correct and this is a core issue
